src/main/java/mods/railcraft/api/charge/CapabilityCharge.java

```diff
@@ -0,0 +1,57 @@
+/*------------------------------------------------------------------------------
+Copyright (c) CovertJaguar, 2011-2020
+
+This work (the API) is licensed under the "MIT" License,
+see LICENSE.md for details.
+-----------------------------------------------------------------------------*/
+
+package mods.railcraft.api.charge;
+
+import mods.railcraft.battery.SimpleBattery;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.nbt.INBT;
+import net.minecraft.util.Direction;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.common.capabilities.Capability.IStorage;
+import net.minecraftforge.common.capabilities.CapabilityInject;
+import net.minecraftforge.common.capabilities.CapabilityManager;
+
+/**
+  * A Capability that can be attached to TE, Entity, and Itemstacks.
+  * This has been generalized so that you can attatch it to basicaly anything,
+  * instead of trains gobbling it up.
+  *
+  * <p>Created by CovertJaguar on 10/4/2016 for Railcraft.
+  *
+  * @author CovertJaguar (https://www.railcraft.info)
+  */
+public final class CapabilityCharge {
```

Comment:
Rename this to `Capabilities`, then we can just shove all our capability registrations in here.